app/controllers/company_branches_controller.rb

```diff
@@ -0,0 +1,44 @@
+class CompanyBranchesController < ApplicationController
+  before_action :set_current_branch, only: %i[edit update destroy]
```

Comment:
Why is this done?
so that the administrator only see their associated branches?